api/index.js

```diff
@@ -66,7 +66,7 @@ const findBirths = (searchFields, accessToken) => {
     : findByNameDOB(searchFields, accessToken);
 };
 
-const userActivityReport = (accessToken, from, to, userFilter) => {
```

Comment:
Is that needed? Seems a strange thing for it to complain over.

Reply:
Yeh apparently we've turned the complexity up to `11` and the linter is losing its tiny little mind!

Reply:
Fair enough.